src/heightmap.cpp

```diff
@@ -22,6 +22,47 @@
 
 #include "safeguards.h"
 
+/**
+ * Maximum number of pixels for one dimension of a heightmap image.
+ *
+ * At some point the number of pixels in a heightmap becomes larger compared to
+ * the pixels that will be taken for the map. With this limit already 75% of the
+ * pixels would be discarded for the largest image and the largest map size.
```

Comment:
I read this 5 times now, and I am still not sure what it says.

"that will be taken for the map" is not something I understand :(

I also don't understand what "75% of the pixels would be discarded" means.

Sorry ..

Reply:
If the image is 8192x8192 pixels, and the size of the map is 4096x4096 then only 25% of the pixels of the image end up being used for the height of the map. The other 75% are discarded.
I'll try to figure out some better wording tomorrow when I've hopefully have somewhat forgotten what I have written, so it becomes clearer what is unclear to others. But I guess map being used interchangeably for the map-array and height-map, together with a parameter `map` in the ReadHeightmap* functions will cause a lot of confusion and headaches.